### PR TITLE
added requirements section

### PR DESCRIPTION
**Pull request by djosborne**
A lot of folks at Mesoscon were interested in "The Catch" with using Calico - what requirements need to be fulfilled on any system to get up and running, and what kind of issues are they going to run into down the line if they commit to using Calico on their large deployments. 

This PR is just a first-draft of addressing these concerns from the start to convince more users to commit to Calico.

### CODE REVIEW

**Comment by lxpollitt** [comment]
Cool.  I would like to review carefully before we publish this.  I had a quick skim and would like to tweak a couple of things, plus think about what more we should be saying to help people.  But I'm unlikely to have time to do that today.

**Comment by tomdee** [comment]
@djosborne @lxpollitt Any progress on this PR?

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ the etcd datastore to define and apply network and security policy to the
 containers you create. Using `calicoctl`, you can provision Calico nodes, 
 endpoints, and define and manage a rich set of security policy. 
 
+## Deployment Requirements
+Calico is extremely versatile - as previously mentioned, it runs its all  core processes in a docker container, truly allowing it to "run anywhere". However, besides a working docker installation, there are a few requirements each host must fulfill in order to succesfully deploy Calico, e.g. IPv4 Forwarding enabled, and ip6_tables kernel modules. These requirements can easily be checked by downloading the calicoctl binary onto the host system and running `calicoctl checksystem`. This will report any configuration issues with the host, and any other dependency failures.
```
Comment (alexwlchan) [inline README.md]: /s/its all/all its

**Comment by tomdee** [comment]
@djosborne @lxpollitt Any progress on this PR?

**Comment by djosborne** [comment]
I browsed through Calico documentation yesterday to get a better idea of whether or not this PR is still relevant. Tim's documentation PR covers what a basic calico cluster layout looks like, and touches on requirements a bit. I think it may make this unnecessary. But then again, having it on the front page more succinctly still seems desirable. @lxpollitt do you have an opinion one way or the other?

**Comment by TrimBiggs** [comment]
I plan to include information similar to this paragraph in my CalicoDocker/LearnCalico documentation in PR #484 that @djosborne mentioned.  Depending on the structure changes I am adding, this may end up on the front page of calico-docker or in a separate document that should be directly linked to from the README.

**Pull request closed without merging**